### PR TITLE
chore(frontend): hide Remote Repository option on the Advanced search page

### PR DESCRIPTION
**Pull request by edalex-yinzi**
##### Checklist

- [x] the [contributor license agreement][] is signed
- [x] commit message follows [commit guidelines][]
- [ ] tests are included
- [x] screenshots are included showing significant UI changes
- [ ] documentation is changed or added

##### Description of change
Hide Remote Repository option on the Advanced search page.

image

[contributor license agreement]: https://www.apereo.org/node/676
[commit guidelines]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

**Comment by PenghaiZhang** [comment]
Hi @edalex-yinzi , on your screenshot, you were on normal search mode, but the remote search was hidden.  We only want to hide the remote search selector when we are on advanced search mode.

File: react-front-end/tsrc/search/SearchPage.tsx [hunk 1]
```diff
@@ -882,7 +882,7 @@ const SearchPage = ({ updateTemplate, advancedSearchId }: SearchPageProps) => {
           urlGeneratorForMuiLink={buildSelectionSessionRemoteSearchLink}
         />
       ),
-      disabled: false,
+      disabled: true,
```
Comment (PenghaiZhang) [inline react-front-end/tsrc/search/SearchPage.tsx:885]: I think whether to disable should depend on the search page mode such as  `searchPageModeState.mode === "advSearch"`

Comment (edalex-ian) [inline react-front-end/tsrc/search/SearchPage.tsx:885]: Eeeck! This will _always_ hide it. Should be done like the Collections selector above.

**Review by edalex-ian — CHANGES_REQUESTED** [review]

**Review by PenghaiZhang — CHANGES_REQUESTED** [review]

**Comment by edalex-yinzi** [comment]
Sorry, I misunderstood the ticket. :disappointed: Now it has been fixed.

**Comment by edalex-yinzi** [comment]
Normal: 

image

Adv search:
image

**Comment by PenghaiZhang** [comment]
We need to reopen this PR to target to release branch.

**Comment by edalex-ian** [comment]
Oh rats, I guess you need to repoint them before deleting the branch. :disappointed:

**Pull request closed without merging**